docs/reference/release-policy.md

```diff
@@ -28,18 +22,18 @@ A cluster usually consists of:
 * Tasklist
 * Optimize
 
-For components outside of the cluster, we release new versions continuously and update customers to the latest version automatically whenever it is ready to be shipped.
+For components **outside of the cluster**, we release new versions continuously and update customers to the latest version automatically whenever it is ready to be shipped.
 
 These components include:
 
 * Modeler (Web)
 * Connectors
 * Console
 
-For components inside a cluster, Camunda provides two channels for provisioning and follows above release policy:
+For components inside a **cluster**, Camunda provides two channels for provisioning and follows above release policy:
```

Comment:
What do we mean by "follows above release policy" here? Do we mean follows **the** above release policy?